arelle/plugin/validate/ESEF/resources/authority-validations.json

```diff
@@ -0,0 +1,278 @@
+                ["http://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor-gen-en.xml"],
+            "https://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor.xsd":
+                ["https://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor-gen-en.xml"],
+            "https://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor.xsd":
+                ["https://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor-gen-en.xml"]
+        },
+        "formulaRunIDs": null
+    },
+    "AT": {
+        "name": "Austria",
+        "reportPackageMaxMB": "100",
+        "reportPackageMeasurement": "zipped"
+    },
+    "BE": {
+        "name": "Belgium",
+        "reportPackageMaxMB": "50",
+        "reportPackageMeasurement": "zipped"
+    },
+    "BG": {
+        "name": "Bulgaria"
+    },
+    "CY": {
+        "name": "Cyprus"
+    },
+    "CZ": {
+        "name": "Czechia"
+    },
+    "DE": {
+        "name": "Germany"
+    },
+    "DK": {
+        "name": "Denmark",
+        "reportPackageMaxMB": "2000",
+        "reportPackageMeasurement": "unzipped",
+        "ixTargetUsage": "allowed"
+    },
+    "EE": {
+        "name": "Estonia",
+        "reportPackageMaxMB": null
+    },
+    "EL": {
+        "name": "Greece"
+    },
+    "ES": {
+        "name": "Spain"
+    },
+    "FI": {
+        "name": "Finland"
+    },
+    "FR": {
+        "name": "France",
+        "reportPackageMaxMB": "40",
+        "reportPackageMeasurement": "zipped"
+    },
+    "HR": {
+        "name": "Croatia"
+    },
+    "HU": {
+        "name": "Hungary"
+    },
+    "IE": {
+        "name": "Ireland"
+    },
+    "IS": {
+        "name": "Iceland"
+    },
+    "IT": {
+        "name": "Italy"
+    },
+    "LI": {
+        "name": "Liechtenstein"
+    },
+    "LT": {
+        "name": "Lithuania"
+    },
+    "LU": {
+        "name": "Luxembourg"
+    },
+    "LV": {
+        "name": "Latvia"
+    },
+    "MT": {
+        "name": "Malta",
+        "reportPackageMaxMB": "25",
+        "reportPackageMeasurement": "zipped"
+    },
+    "NL": {
+        "name": "Netherlands",
+        "reportPackageMaxMB": "100",
+        "reportPackageMeasurement": "unzipped",
+        "extensionAbstractContexts": "allowed",
+        "reportFileNamePattern": "{base}-{date}.xhtml",
+        "reportFileNameRegex": "(.{1,})-[0-9]{4}-[0-9]{2}-[0-9]{2}[.]xhtml$",
+        "extensionElementsAnchoring": "include abstract",
+        "dimension-default-arc-ELRs": "990000 only",
+        "LC3AllowCapitalsInWord": false
+    },
+    "NO": {
+        "name": "Norway"
+    },
+    "PL": {
+        "name": "Poland"
+    },
+    "PT": {
+        "name": "Portugal"
+    },
+    "RO": {
+        "name": "Romania"
+    },
+    "SE": {
+        "name": "Sweden",
+        "reportPackageMaxMB": "100",
+        "reportPackageMeasurement": "zipped"
+    },
+    "SI": {
+        "name": "Slovenia",
+        "reportPackageMaxMB": null
+    },
+    "SK": {
+        "name": "Slovakia"
+    },
+    "UK": {
```

Comment:
It is my understanding that UKSEF is its own thing and does not relate to ESEF. So it is entirely possible that the UK continues to accept both UKSEF and ESEF filings in perpetuity. So it seems to make sense that this key would be more specific to `UKSEF` and then we keep `UK` for anything that is directly ESEF related. This key would remain empty for now since we do not have any direct ESEF validation specifics.

Reply:
Agree, XII filing index already has "GB" filings, will rename this to UKSEF.